src/data/index.js

```diff
@@ -0,0 +1,8 @@
+export const fakeData = {
```

Comment:
This too?
Ideally all of the fake data should be replaced by now

Reply:
I will gradually replace these fake data after I can GET and POST playlists under a offering